recipes/rmlui/3.3/conanfile.py

```diff
@@ -65,17 +65,17 @@ def _configure_cmake(self):
 
             cmake_version_output = StringIO()
             self.run("%s --version" % self._cmake._cmake_program, output=cmake_version_output)
-            cmake_version = Version(cmake_version_output.getvalue().split('\n', 1)[0].rsplit(' ', 1)[-1])
+            cmake_version = tools.Version(cmake_version_output.getvalue().split('\n', 1)[0].rsplit(' ', 1)[-1])
 
             self._cmake.definitions["BUILD_LUA_BINDINGS"] = self.options.with_lua_bindings
             self._cmake.definitions["BUILD_SAMPLES"] = False
             self._cmake.definitions["DISABLE_RTTI_AND_EXCEPTIONS"] = not self.options.enable_rtti_and_exceptions
             self._cmake.definitions["ENABLE_PRECOMPILED_HEADERS"] = cmake_version >= "3.16.0"
```

Comment:
This is not very useful with Conan. It will delete the build folder by default and it's extremely unlikely to be used....

I would just add a `builtd_)requires` for the latest cmake and add a comment.... avoid parsing the output

Reply:
This one deserves just a tiny bit of a background: I wanted to use just CMake.get_version() at first, because it looked pretty straightforward, but that ended up not working out since isn't always available from PATH. This basically means that the patch I gave is more of a workaround than anything else, which I'm never really a fan of.

Your suggestion, however, sounds miraculously easy and very straightforward. Thanks for pointing me in the right direction!